webapp/src/App.js

```diff
@@ -140,7 +140,7 @@ function AppRoutes() {
             redirectIfUnauthed,
             redirectIfUnboarded,
             withScreenLoaderMount(),
-            withMetatags({ title: t("mobility:title") }),
+            withMetatags({ title: t("titles:mobile") }),
```

Comment:
These refer to different conceptual strings, don't change things like this for the sake of it. Ie, even if we went with 'Mobile' for nav there's no reason to change it for page titles.